docs/user_guide/monitoring.md

```diff
@@ -0,0 +1,114 @@
+# Monitor Dragonfly with Prometheus
```

Comment:
In these document, I think we should first tell users that which specific metrics supernode and dfdaemon support. Use a table to list the detailed metrics?

Reply:
Yes, I will add a link to another docs which contains all the metrics information.